README.md

```diff
@@ -150,32 +150,10 @@ Excecute the notebooks in Binder:
 
 
 # Setup R and Python environments
-## R
-Run the following commands in R to install all required packages
-```
-if (!require(vioplot)) install.packages('vioplot')
-if (!require(corrplot)) install.packages('corrplot')
-if (!require(gmodels)) install.packages('gmodels')
-if (!require(matrixStats)) install.packages('matrixStats')
-
-if (!require(lmPerm)) install.packages('lmPerm')
-if (!require(pwr)) install.packages('pwr')
-
-if (!require(FNN)) install.packages('FNN')
-if (!require(klaR)) install.packages('klaR')
-if (!require(DMwR)) install.packages('DMwR')
-
-if (!require(xgboost)) install.packages('xgboost')
-
-if (!require(ellipse)) install.packages('ellipse')
-if (!require(mclust)) install.packages('mclust')
-if (!require(ca)) install.packages('ca')
-```
 
-## Python
-We recommend to use a conda environment to run the Python code.
+We recommend to use a conda environment to run the Python and R code.
 ```
-conda create -n sfds python
+conda create -n sfds
 conda activate sfds
 conda env update -n sfds -f environment.yml
 ```
```

Comment:
We should probably still have a list of the dependencies or point the user to the `environment.yml` file for that list.

Reply:
I've added a list of dependencies to the README.md file, along with adding comments to the shell script.